src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,12 @@
+* {
```

Comment:
you have two files reset.scss in your project, so you shold use only one and remove another